### ISSUE TITLE
Sync not working if combined with reviewing sync operations

### ISSUE THREAD

**Issue by vic-t** [lukasz-wronski/vscode-ftp-sync#236]
So I'm running a sync, remote to local. The tool asks if I want to review the changes it is going to make, I confirm which opens a *.json file with the following content. 

`{
    "_readMe": "Review list of sync operations, then use Ftp-sync: Commit command to accept changes",
    "_warning": "This file should not be saved, reopened review file won't work!",
    "filesToUpdate": [
        "index.html"
    ],
    "filesToAdd": [],
    "dirsToAdd": [],
    "filesToRemove": [],
    "dirsToRemove": []
}`

In my understanding, if I want for the operation to continue, I have to call "ftp-sync: commit" without doing anything else. However, if I do that, I get an error message, complaining that I should run a sync first. 

> ftp-sync: no operations list to commit. Run sync first. 

Is this a bug or am I doing something wrong?

**Comment by martinbryant** [comment]
I am receiving the same error when I try this

**Comment by patrix2001** [comment]
Yes, I am also seeing this issue. Even though it appears we are doing the correct thing (eg. Running a sync -> viewing the json file -> running "ftp-sync: commit"

**Comment by mborn319** [comment]
Yep, this is broken on my machine as well.

**Comment by mborn319** [comment]
If I'm reading this right, the error is thrown because the json file does not have a `syncOptions` json key. https://github.com/lukasz-wronski/vscode-ftp-sync/blob/8cbb51a8a0bf0cc1d9e0adbc16160e1310dc2ed1/modules/commit-command.js#L16

This appears to be set here: https://github.com/lukasz-wronski/vscode-ftp-sync/blob/8cbb51a8a0bf0cc1d9e0adbc16160e1310dc2ed1/modules/sync-command.js#L32
